sites/kit.svelte.dev/package.json

```diff
@@ -20,7 +20,6 @@
 		"shiki-twoslash": "^3.0.2",
 		"svelte": "^3.43.0",
 		"typescript": "~4.6.2",
-		"vite": "^2.9.0",
```

Comment:
this needs to be here otherwise the docs won't build (because shiki-twoslash needs to be able to see the types)

Reply:
ok. added it back